Admin ui features - Mariann- ready to review.md (#72)
SenseNet/docs.sensenet.com: content/guides/search.md

metaTitle: "sensenet admin-ui - Search"
metaDescription: "sensenet admin-ui - Search"
---

> # Search

Reviewer: ```suggestion
```

> In sensenet one of the most important and sophisticated features is **Search**.

Reviewer: ```suggestion
In sensenet one of the most important and sophisticated feature is **Search**.
```

As everything is a content in sensenet your query results cannot contain only files, but also users, tasks, workspaces ect.

> It is possible to search even in uploaded documents.

Reviewer: itt most magában a szövegben keresésrre gondolsz?

> ![search](/content/concepts/img/search.gif)

Reviewer: ```suggestion
![search](../concepts/img/search.gif)
```

It is possible to extract all relevant terms from a text, filtering stopwords (like "a", "an", "and", "are", "as", "at", "be", "but", "by", "for", "if", "in") etc. In case your document contains the following text: „Writing Sentences” and your query text is „writing”. The original text can be found even if the query word typed in and the word in the original text do not match exactly char-by-char. You can get relevant search results even if you have a typo in your query or a different verbal tense.
_Full Text Search_, _Fuzzy Search_, _Proximity Search_ and _Boosting_ make your query results more complete.

> To learn more about the methology behind go to [link search].

Reviewer: ```suggestion
To learn more about the methology behind this feature, check [search article](/content/concepts/basics/04-search) in the concept docs.
```
